src/components/TreeMenu/TreeMenu.jsx

```diff
@@ -29,7 +29,7 @@ export const TreeMenu = ({
 							{hasSearch &&
 								<Input
 									onChange={e => search(e.target.value)}
-									placeholder={t('General|Search')}
+									placeholder={t('General|Search by phrase')}
```

Comment:
"Search" seems to me ok for me - it is universal

Reply:
@ateska Yes, however it brought the confusion for users, since this searches only within the treemenu (and when you have search also in some other component on the screen, like fulltext search in library, it can be misleading). So we came up with this name on a review a while ago. Should I still change it to simple `Search` ?